packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -8,5 +8,6 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
-  "cancel_sfdx_command_text": "SFDX: Cancel Active Command"
+  "cancel_sfdx_command_text": "SFDX: Cancel Active Command",
+  "force_apex_class_create": "SFDX: Create Apex Class"
```

Comment:
@ruthemmanuelle It seems weird to include the articles "a/an" in the command names since it looks like they're excluded from most (if not all, I skimmed thru all of them) command names. For example, for Git commands the names are "Git: Create Branch" rather than "Git: Create a Branch". Seems like an unspoken convention of VSCode commands but thought I'd point it out.

Also didn't mean to delete the image. I've reattached
![screen shot 2017-08-17 at 11 33 06 am](https://user-images.githubusercontent.com/4503716/29430663-16e814c2-834a-11e7-8c8f-87319db7bba7.png)
 it.